packages/babel-traverse/src/path/evaluation.ts

```diff
@@ -430,7 +430,11 @@ function evaluateQuasis(
   let str = "";
 
   let i = 0;
-  const exprs = path.get("expressions");
+  const exprs: Array<NodePath<t.Node>> = path.isTemplateLiteral()
+    ? path.get('expressions')
+    : (path as NodePath<t.TaggedTemplateExpression>)
+        .get('quasi')
+        .get('expressions');
```

Comment:
```suggestion
  const exprs = path.isTemplateLiteral()
    ? path.get("expressions")
    : path.get("quasi.expressions");
```
Will this work fine?

Reply:
This will work fine, but I’m not sure if the types will work correctly with it.